device/mellanox/x86_64-mlnx_msn3800-r0/Mellanox-SN3800-D112C8/sai.profile

```diff
@@ -1 +1 @@
-SAI_INIT_CONFIG_FILE=/usr/share/sonic/hwsku/sai_3800_112x50g_8x100g.xml
```

Comment:
Kindly keep the name as before as that's a convention followed for other SKU's as well. You can check [this ](https://github.com/Azure/sonic-buildimage/blob/master/device/mellanox/x86_64-mlnx_msn2700-r0/Mellanox-SN2700-D48C8/sai_2700_48x50g_8x100g.xml).
It can be used to identify the file in syncd docker

Reply:
addressed in the next commit.